### PR TITLE
JBIDE-27582: reworked Quarkus test given new quarkus project

### PR DESCRIPTION
**Pull request by olkornii**
Signed-off-by: olkornii <olkornii@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by olkornii** [comment]
https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Quality/job/Integration-tests/job/quarkus-itests/313/

**Comment by jkopriva** [comment]
testPR

**Comment by olkornii** [comment]
Ready for merge.

File: itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/content/assistant/ApplicationPropertiesNewExtensionContentAssistTest.java [hunk 1]
```diff
@@ -65,14 +65,14 @@ public void testContentAssistOldNew() {
...
 		ExtensionsView ev = new ExtensionsView();
 		ev.open();
-		ev.getExtension("RESTEasy").select();
+		ev.getExtension("SmallRye OpenAPI").select();
 		new ContextMenuItem("Install extension").select();
 		new WaitWhile(new JobIsRunning(), TimePeriod.VERY_LONG);
 
 		try {
 			String pomContent = InstallQuarkusExtensionTest
 					.readFile(WORKSPACE + "/" + NEW_EXTENSION_PROJECT_NAME + "/pom.xml");
```
Comment (jkopriva) [inline itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/content/assistant/ApplicationPropertiesNewExtensionContentAssistTest.java:74]: Maybe its time to move readfile function to AbstractQuarkusTest

File: itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/RunProjectWithDebugTest.java [hunk 2]
```diff
@@ -172,4 +182,50 @@ public String description() {
 		new ShellMenuItem("Run", "Resume").select();
 		new WaitUntil(new ConsoleHasText(consoleView, "hello commando"), TimePeriod.LONG);
 	}
+
+	private static void changeProject() {
+		File file = new File(FILE_PATH).getAbsoluteFile();
+
+		String st = "";
+		StringBuilder bld = new StringBuilder();
```
Comment (jkopriva) [inline itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/RunProjectWithDebugTest.java:190]: please move this to AbstractTest class

File: itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/RunProjectWithDebugTest.java [hunk 3]
```diff
@@ -172,4 +182,50 @@ public String description() {
...
+		String newProject = bld.toString();
+
+		if (!newProject.equals("")) {
+			ProjectItem exampleResource = new ProjectExplorer().getProject(PROJECT_NAME).getProjectItem(RESOURCE_PATH)
+					.getProjectItem(ORG_ACME).getProjectItem(EXAMPLE_RESOURCE);
+
+			exampleResource.open();
+			TextEditor ed = new TextEditor(EXAMPLE_RESOURCE);
+
+			new WaitUntil(new AbstractWaitCondition() {
```
Comment (jkopriva) [inline itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/RunProjectWithDebugTest.java:209]: you cannot use condition ShellIsActive?

**Review by jkopriva — CHANGES_REQUESTED** [review]

**Comment by olkornii** [comment]
test PR

**Comment by olkornii** [comment]
testPR

File: itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTest.java [hunk 4]
```diff
@@ -113,10 +114,68 @@ public static void createNewFile(String projectName, String fileName, String fil
 		WizardNewFileCreationPage page = new WizardNewFileCreationPage(newFileDialog);
 		page.setFileName(fileName);
 		page.setFolderPath(projectName + "/" + filePath);
-		newFileDialog.finish();
+		try {
+			newFileDialog.finish();
+		} catch (org.eclipse.reddeer.common.exception.WaitTimeoutExpiredException e) { // sometimes node.js warning
```
Comment (jkopriva) [inline itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTest.java:119]: is this related to this issue? https://issues.redhat.com/browse/JBIDE-27432

**Pull request merged**